libs/vue-plugin/package.json

```diff
@@ -26,6 +26,9 @@
     "url": "https://github.com/ZachJW34/nx-plus/issues"
   },
   "homepage": "https://github.com/ZachJW34/nx-plus/tree/master/libs/vue-plugin#readme",
+  "scripts": {
+    "postinstall": "node src/scripts/postinstall.js"
```

Comment:
This feels like a breach of trust between package author and user. Users don't expect packages to change the contents of other packages. To be clear, my concern is when `postinstall.js` is ran, not the script itself. We should consider replacing this with documentation on adding the `postinstall` npm script in the user's `package.json`.

Reply:
Maybe add the npm script it in the app schematic?

Reply:
I agree. It feels malicious, or at least I can imagine a shady package using a postinstall in their package to do something malicious. I like the idea of adding it to the user's package.json via schematic

Reply:
Ok. Add it with the schematic and if for whatever reason it can't be added, maybe `postinstall` already exists, then fallback to logging a warning. Since we have no docs right now, don't bother adding docs. I'll add it when I get to the docs.